tsconfig.nextconfig.json

```diff
@@ -1,15 +1,14 @@
 {
   "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["esnext"],
+    "noEmit": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "resolveJsonModule": true,
-    "module": "commonjs",
-    "outDir": ".",
-    "target": "es5",
-    "lib": [
-      "es2017"
-    ]
+    "removeComments": true,
```

Comment:
Why is `removeComments` here `true`, compared to react-prime where you explicitly set it to `false`?

Reply:
This is the tsconfig for the nextconfig file, not the source code